File: src/main/java/com/bpzj/auto/deploy/listeners/MyProjectManagerListener.java
package com.bpzj.auto.deploy.listeners;

import com.bpzj.auto.deploy.services.MyProjectService;
import com.intellij.openapi.project.Project;
import com.intellij.openapi.project.ProjectManagerListener;
import org.jetbrains.annotations.NotNull;

public class MyProjectManagerListener implements ProjectManagerListener {
    @Override
    public void projectOpened(@NotNull Project project) {
        project.getService(MyProjectService.class);
    }
}

File: src/main/kotlin/com/github/bpzj/autodeploy/listeners/MyProjectManagerListener.kt
package com.github.bpzj.autodeploy.listeners

import com.intellij.openapi.project.Project
import com.intellij.openapi.project.ProjectManagerListener
import com.github.bpzj.autodeploy.services.MyProjectService

internal class MyProjectManagerListener : ProjectManagerListener {

    override fun projectOpened(project: Project) {
        project.getService(MyProjectService::class.java)
    }
}
